lib/backend/k8s/conversion/constants.go

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2017 Tigera, Inc. All rights reserved.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package conversion
+
+const (
+	NamespaceLabelPrefix       = "pcns."
+	Orchestrator               = "k8s"
```

Comment:
For better readability I think this should be `OrchestratorKubernetes` or similar.

Probably fine for now since there's only one value, but we might eventually have `OrchestratorMesos` etc

Reply:
Well if we keep the constant here then it is scoped by teh KDD package

If you want me to move to apiv2 then I'll Add a set of values and make this OrchestratorKubernetes

Reply:
just say the word and i'll do either

Reply:
Actually it should be in apiv2!  I'll move it!